backends/gpu/lib/kernels/blas_kernels.cc

```diff
@@ -213,23 +213,24 @@ static Error BlasTrsmBatch(
     wrapper::Pointer<T*> b_array_ptr(b_array, platform);
     auto cast_alpha_ptr = static_cast<wrapper::Pointer<const T>>(*alpha_ptr);
 
-    return wrapper::CublasTrsmBatched(
-        *current, handle.get(), side_mode,
-        wrapper::BlasFillMode::FromOpaqueValue(*fillMode),
-        wrapper::BlasOperation::FromOpaqueValue(*trans),
-        wrapper::BlasDiagType::FromOpaqueValue(*diagType), m, n, cast_alpha_ptr,
-        a_array_ptr, heightA, b_array_ptr, heightB, batchCount);
+    
+    return wrapper::RocblasTrsmBatched(
```

Comment:
I want to suggest that you add a BlasTrsmBatched to blas_wrapper.h/cc and call that from here. But complex types are different for CUDA and ROCm. So likely we want to take care of the `TODO` above CublasTrsmBatched and provide a type-punned entry point instead (plus an platform abstracted version).